FusionIIIT/applications/ps1/models.py

```diff
@@ -38,4 +38,20 @@ class StockEntry(models.Model):
     bill=models.FileField(blank=False)
 
     class Meta:
-        db_table = 'StockEntry'
\ No newline at end of file
+        db_table = 'StockEntry'
+
+class GlobalStockAdmin(models.Model):
+    s_no = models.IntegerField(null=False)
+    item_name = models.CharField(max_length=250, blank=False)
+    department = models.CharField(max_length=100, blank=False)
+    section = models.CharField(max_length=100)
+    floor_no = models.IntegerField(null=False)
+    rooms = models.IntegerField(max_length=100)
+    year = models.IntegerField(null=False)
+    item_id = models.IntegerField(null=False)
+    contd = models.BooleanField
```

Comment:
the syntax for a boolean field is this 
`my_boolean_field = models.BooleanField(default=False)`
also give meaningful names to fields